apps/authentication/mfa/face.py

```diff
@@ -54,4 +51,8 @@ def can_disable(self) -> bool:
 
     @staticmethod
     def help_text_of_enable():
-        return _("Frontal Face Recognition")
+        return _("Bind face to enable")
+
+    @staticmethod
+    def help_text_of_disable():
+        return _("Unbind face to disable")
```

Comment:
Sorry, but I need more information on which code block you're referring. Could you please specify the specific parts of the Python code that needs checking?